apps/src/templates/sectionAssessments/MultipleChoiceDetailsDialog.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/no-danger */
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import Button from '@cdo/apps/templates/Button';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import i18n from "@cdo/locale";
+import DialogFooter from "@cdo/apps/templates/teacherDashboard/DialogFooter";
+import processMarkdown from 'marked';
+import renderer from "@cdo/apps/util/StylelessRenderer";
+import {getCurrentQuestion} from "./sectionAssessmentsRedux";
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  instructions: {
+    marginTop: 20
+  }
+};
+
+class MultipleChoiceDetailsDialog extends Component {
+  static propTypes = {
+    isDialogOpen: PropTypes.bool.isRequired,
+    closeDialog: PropTypes.func.isRequired,
+    questionText: PropTypes.string,
+  };
+
+  render() {
+    // Questions are in markdown format and should not display as plain text in the dialog.
+    const renderedMarkdown = processMarkdown(this.props.questionText, { renderer });
```

Comment:
This is our current standard for displaying markdown in react. Elijah's working on something safer next sprint.

Reply:
erm... I meant thanks for the heads up on the i18n issue.